client/src/Pages/UserProfile.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from "react";
+import Media from "../Components/Media/Media";
+import usersData from "../Data/profileData.json";
+
+import "../Styles/UserProfile.css";
+
+var usersList = usersData.users;
+var statsList = usersData.stats;
+
+function Profile() {
+  return (
+    <div className="container home">
+      <div
+        className="row justify-content-center"
+        style={{ marginBottom: "40px", marginTop: "100px" }}
+      >
+        <div className="mb-5 col-12 col-md-4  ">
+          {usersList.map((post) => {
+            return (
+              <div key={post.id} className="name-insti">
+                <h3>{post.name}</h3>
+                <h5>{post.institute}</h5>
+              </div>
+            );
+          })}
+          {usersList.map((post) => {
+            return (
+              <div key={post.id} className=" mt-5 mb-2 profile-pic  ">
+                <img
+                  className="mypic"
+                  src= {post.profile_pic}
+                  alt="profile pic"
+                />
+              </div>
+            );
+          })}
+
+          
+
+          <div className="follow3 col-auto">
+            <span>
+              <button className="follow1">
+                <a className="links" href="#">
+                  Following
+                </a>
+              </button>
+              <button className="follow2">
+                <a className="links" href="#">
+                  Followers
+                </a>
+              </button>
+            </span>
+          </div>
```

Comment:
Can we have an extra information block to show the number of followers/following?

Reply:
sorry I didn't get this. Like a button "extra info" which shows followers/following?